TodoAppWithSpringboot/src/main/java/com/webonise/exceptioncontroller/UserExceptionController.java

```diff
@@ -0,0 +1,40 @@
+package com.webonise.exceptioncontroller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import com.webonise.exception.EmailAlreadyExistException;
+import com.webonise.exception.UserNotExistException;
+import com.webonise.exception.UserNotInitializedException;
+import com.webonise.exception.UsernameAlreadyExistException;
+import com.webonise.exception.UsernameNotInitializedException;
+
+@ControllerAdvice
+public class UserExceptionController {
+	
```

Comment:
could be only one GlobalException class, and creating Generic Exceptions based on HttpCode and setting specific error messages would be better